Lesson4.ipynb

```diff
@@ -6,7 +6,10 @@
    "source": [
     "## Lesson 4 Overview\n",
     "\n",
-    "### TODO"
+    " * Read CSV file\n",
+    " * List contents of the CSV file\n",
+    " * Redo lesson 3 questions after reading CSV\n",
```

Comment:
Please remove the point on "Redo lesson 3 questions after reading CSV". Don't believe this was covered in Lesson 4.